AIDevGallery/Utils/AIToolkitHelper.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AIDevGallery.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AIDevGallery.Utils;
+
+internal static class AIToolkitHelper
+{
+    private static Dictionary<AIToolkitAction, ToolkitActionInfo> aiToolkitActionInfos = new()
+    {
+        { AIToolkitAction.FineTuning, new ToolkitActionInfo() { DisplayName = "Fine Tuning", QueryName = "open_fine_tuning" } },
+        { AIToolkitAction.PromptBuilder, new ToolkitActionInfo() { DisplayName = "Prompt Builder", QueryName = "open_prompt_builder" } },
+        { AIToolkitAction.BulkRun, new ToolkitActionInfo() { DisplayName = "Bulk Run", QueryName = "open_bulk_run" } },
+        { AIToolkitAction.Playground, new ToolkitActionInfo() { DisplayName = "Playground", QueryName = "open_playground" } }
+    };
+
+    public static Dictionary<AIToolkitAction, ToolkitActionInfo> AIToolkitActionInfos
+    {
+        get { return aiToolkitActionInfos; }
+    }
+
+    public static string CreateAiToolkitDeeplink(AIToolkitAction action, ModelDetails modelDetails)
```

Comment:
Do these maybe make the methods on this class as extension methods? Would make using them a little bit nicer. Just a suggestion.

Reply:
Now that you changed the other methods to extension methods, this one is missing 😅